### PR TITLE
added cineSync to supported applications

### PR DESCRIPTION
**Pull request by cueball66**

### CODE REVIEW

File: docs/guides/using_ocio/compatible_software.rst [hunk 1]
```diff
@@ -387,6 +387,16 @@ Documentation :
 - `VRayTexOCIO <https://docs.chaosgroup.com/display/VRAY4MAYA/VRayTexOCIO>`__
 
 
+cineSync (ftrack)
```
Comment (hodoulp) [inline docs/guides/using_ocio/compatible_software.rst:390]: The list follows the alphabetic order. Could move the paragraph?

File: docs/site/homepage/data/en/supported_apps.yml [hunk 2]
```diff
@@ -192,6 +192,11 @@ supported_apps:
       content : V-Ray 4 and up.
       link : "https://chaosgroup.com/"
 
-
+    # portfolio item loop
```
Comment (hodoulp) [inline docs/site/homepage/data/en/supported_apps.yml:195]: The list follows the alphabetic order. Could move the paragraph?

**Review by hodoulp — CHANGES_REQUESTED** [review]
Thanks @cueball66 for the contribution to the project.
I only have one comment concerning the alphabetical order of the tool list.

File: docs/guides/using_ocio/compatible_software.rst (hunk 1 above)
Comment (cueball66) [reply docs/guides/using_ocio/compatible_software.rst]: This should be fixed now

**Review by hodoulp — APPROVED** [review]

**Review by michdolan — APPROVED** [review]
Thanks!

**Pull request merged**